go-runtime/compile/testdata/two/two.go

```diff
@@ -15,10 +15,23 @@ const (
 	Green TwoEnum = "Green"
 )
 
-//ftl:data
+//ftl:enum
+type TypeEnum interface{ typeEnum() }
+
+type Scalar string
+
+func (Scalar) typeEnum() {}
+
+type List []string
+
+func (List) typeEnum() {}
+
+//ftl:data export
```

Comment:
in addition to explicit, we should also test transitive exporting

Reply:
ahhh good call - adding that now

Reply:
done